src/components/Empty/Empty.component.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFrog, faHeartBroken } from '@fortawesome/free-solid-svg-icons';
+
+const Container = styled.div`
+  border: 1px solid ${(props) => props.theme.divider};
+  width: 100%;
+  height: 250px;
+  background-color: ${(props) => props.theme.body};
+  word-wrap: break-word;
+  box-sizing: border-box;
+  text-align: center;
+  padding-top: 50px;
+`;
+
+const Title = styled.h1`
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 13px;
+  color: ${(props) => props.theme.text};
+`;
+
+function Empty(props) {
+  const { container } = props;
+  const isFav = container === '2';
```

Comment:
I recommend changing the strategy of using container 1 or 2, you could use instead constants with words i.e:
`const isFav = container === CONSTANT`

`const CONSTANT = 'constant'`